pyanaconda/ui/tui/hubs/__init__.py

```diff
@@ -61,6 +62,11 @@ def __init__(self, data, storage, payload):
         # we want user input
         self.input_required = True
 
+    @property
+    @abstractmethod
+    def title(self):
+        return _("Default HUB title")
```

Comment:
Just return `None`, please. The property has to be redefined anyway.

Reply:
Sounds good - this is one of the things I was not sure how to do correctly, so thanks a lot for feedback! :)